conda/core/initialize.py

```diff
@@ -979,8 +979,13 @@ def init_fish_user(target_path, conda_prefix, reverse):
     # target_path: ~/.config/config.fish
     user_rc_path = target_path
 
-    with open(user_rc_path) as fh:
-        rc_content = fh.read()
+    try:
+        with open(user_rc_path) as fh:
+            rc_content = fh.read()
+    except FileNotFoundError:
+        rc_content = ''
+    except:
```

Comment:
This is not really necessary.  Uncaught exceptions will always be raised.  Fine to have it here, though.